tuner/examples/simple/simple_tuner.py

```diff
@@ -72,6 +72,11 @@ def main():
     )
     # Remaining arguments come from libtuner
     args = libtuner.parse_arguments(parser)
+    return args
+
+
+def main():
```

Comment:
We use type hints in the tuner
```suggestion
def main() -> None:
```